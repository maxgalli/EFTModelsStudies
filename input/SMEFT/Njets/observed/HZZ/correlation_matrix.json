{
    "r_Njets_0": {
        "r_Njets_0": 1.0,
        "r_Njets_1": -0.17871191813241327,
        "r_Njets_2": 0.08792918027826827,
        "r_Njets_3": -0.013628133297734239,
        "r_Njets_4": -0.036048955627587184
    },
    "r_Njets_1": {
        "r_Njets_0": -0.17871191813241327,
        "r_Njets_1": 1.0,
        "r_Njets_2": -0.3093595389070284,
        "r_Njets_3": 0.0748585062654434,
        "r_Njets_4": 0.05070925870364875
    },
    "r_Njets_2": {
        "r_Njets_0": 0.08792918027826827,
        "r_Njets_1": -0.3093595389070284,
        "r_Njets_2": 1.0,
        "r_Njets_3": -0.3041016314311076,
        "r_Njets_4": -0.15942115326319728
    },
    "r_Njets_3": {
        "r_Njets_0": -0.013628133297734239,
        "r_Njets_1": 0.0748585062654434,
        "r_Njets_2": -0.3041016314311076,
        "r_Njets_3": 1.0,
        "r_Njets_4": 0.11490563620468291
    },
    "r_Njets_4": {
        "r_Njets_0": -0.036048955627587184,
        "r_Njets_1": 0.05070925870364875,
        "r_Njets_2": -0.15942115326319728,
        "r_Njets_3": 0.11490563620468291,
        "r_Njets_4": 1.0
    }
}