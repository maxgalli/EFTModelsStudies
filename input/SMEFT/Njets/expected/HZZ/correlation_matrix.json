{
    "r_Njets_0": {
        "r_Njets_0": 1.0,
        "r_Njets_1": -0.15381073729449202,
        "r_Njets_2": 0.07835782229924639,
        "r_Njets_3": -0.007724071782390584,
        "r_Njets_4": 0.009086615731703333
    },
    "r_Njets_1": {
        "r_Njets_0": -0.15381073729449202,
        "r_Njets_1": 1.0,
        "r_Njets_2": -0.3052662670490454,
        "r_Njets_3": 0.09764615032097977,
        "r_Njets_4": 0.007195421286396091
    },
    "r_Njets_2": {
        "r_Njets_0": 0.07835782229924639,
        "r_Njets_1": -0.3052662670490454,
        "r_Njets_2": 1.0,
        "r_Njets_3": -0.40114699533033094,
        "r_Njets_4": 0.11643186963341726
    },
    "r_Njets_3": {
        "r_Njets_0": -0.007724071782390584,
        "r_Njets_1": 0.09764615032097977,
        "r_Njets_2": -0.40114699533033094,
        "r_Njets_3": 1.0,
        "r_Njets_4": -0.3847460336840018
    },
    "r_Njets_4": {
        "r_Njets_0": 0.009086615731703333,
        "r_Njets_1": 0.007195421286396091,
        "r_Njets_2": 0.11643186963341726,
        "r_Njets_3": -0.3847460336840018,
        "r_Njets_4": 1.0
    }
}